### ISSUE TITLE
Dark theme not working

### ISSUE THREAD

**Issue by Martin5001** [ChrisAnd1998/TaskbarX#236]
Hi Chris, the new Configurator UI is really nice, awesome work!
Unfortunately dark theme (#68) does not work in v. 1.5.4.0. I have Windows set to all dark (both apps and system).
Thanks

**Comment by ChrisAnd1998** [comment]
Thx! It only works if you restart the configurator. It will not change while it's running.

**Comment by Martin5001** [comment]
I tried it, even restarted the pc, but it doesn't work. Tried on Windows 10 1909 stable and 2004 preview.

**Comment by ChrisAnd1998** [comment]
@Martin5001 Weird, it's working fine here on 2004

**Comment by ArttuLai** [comment]
I was on 1.5.5.0 (free version from GitHub) and couldn't make the app go dark either. I tried to re-install (while using dark system theme), restart the configurator, restart the PC, but it stays light. I tried switching system theme to light and back to dark and then do the above steps, but I just couldn't get it to work. 

I'm using Surface Laptop 3 with 1909 (build 18363.815).

Windows Store version 1.5.5.0 has working dark mode though, I tried it just now.
